dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -85,6 +92,29 @@ private PageResourceHelper() {
 
     }
 
+    public void saveContent(String pageId, List<PageContainerForm.ContainerEntry> containerEntries) throws DotDataException {
+        List<MultiTree> multiTres = new ArrayList<>();
+
+        for (PageContainerForm.ContainerEntry containerEntry : containerEntries) {
+            int i = 0;
+            List<String> contentIds = containerEntry.getContentIds();
+
+            for (String contentletId : contentIds) {
+                MultiTree mt = new MultiTree().setContainer(containerEntry.getContainerId())
+                        .setContentlet(contentletId)
+                        .setRelationType("LEGACY_RELATION_TYPE")
```

Comment:
We need the UUID of the container passed in here, not the LEGACY_RELATION_TYPE - this will allow us to have the same container on the page more than once.